### PR TITLE
Full Air Mozilla Integration

### PR DESCRIPTION
**Pull request by Nolski**

### CODE REVIEW

File: airmozilla/popcorn/renderer.py [hunk 1]
```diff
@@ -30,20 +31,32 @@ def render_edit(edit_id):
...
-        process_json(edit.data, out=filepath)
-
-    upload_file_name = '%s.webm' % uuid4().hex[:13]
+        process_json(
+            data=edit.data['data'],
+            out=filepath,
+            background_color=edit.data['background']
+        )
+
+    d = datetime.date.today()
```
Comment (peterbe) [inline airmozilla/popcorn/renderer.py]: No, use UTC instead. 
E.g. 
```python
prefix = datetime.datetime.utcnow().strftime('%Y/%m/%d/')
upload_file_name = prefix + uuid4().hex[:13] + '.webm
```
Normally I would use `os.path` but this isn't strictly a directory as S3 is odd.

File: airmozilla/popcorn/renderer.py [hunk 2]
```diff
@@ -66,6 +82,9 @@ def render_edit(edit_id):
         upload_time=int(end - start)
     )
 
+    if verbose:
+        print 'Upload object created with id: %s' % str(upload.id)
```
Comment (peterbe) [inline airmozilla/popcorn/renderer.py]: No need to for `str()` since the `%s` does that for you.

File: airmozilla/popcorn/renderer.py [hunk 3]
```diff
@@ -30,20 +31,32 @@ def render_edit(edit_id):
             os.stat(filepath)[os.stat.st_size] > 0):
         edit.status = PopcornEdit.STATUS_PROCESSING
```
Comment (peterbe) [inline airmozilla/popcorn/renderer.py]: Perhaps add a `if verbose:` on the `filepath` too.

File: airmozilla/popcorn/static/popcorn/js/editor.js [hunk 4]
```diff
@@ -18,8 +18,9 @@ $(document).ready(function () {
   // Initialize the editor with the div id and path to Popcorn Editor.
   PopcornEditor.init('editor', '/static/popcorn/PopcornEditor/editor.html');
   PopcornEditor.listen('save', function (message) {
+    console.log(JSON.stringify(message));
```
Comment (peterbe) [inline airmozilla/popcorn/static/popcorn/js/editor.js]: debug leftovers

**Comment by peterbe** [comment]
r+ with nits

**Comment by Nolski** [comment]
@peterbe Hey, sorry I got caught up at the internet archive meeting. Fixed up your comments, if it looks good feel free to merge.

**Comment by peterbe** [comment]
Can you attach the commit message to a bug?

**Comment by peterbe** [comment]
...and squash please.

**Comment by Nolski** [comment]
@peterbe woops, my bad. All squashed and rebased. Should be good to merge if tests still pass.

**Pull request merged**